### PR TITLE
Add AWS Polly TTS support

### PR DESCRIPTION
**Pull request by markbackman**
Adding AWS Polly TTS service using SSML formatting.

Unfortunately, AWS Polly doesn't provide streaming feedback of the audio produced. It waits until all audio is collected. This makes it a poor fit, at the moment, for conversational AI.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added AWS Polly TTS support.
+
+- Added InputParams to Azure TTS service.
```
Comment (markbackman) [inline CHANGELOG.md:14]: This has been merged already. I forgot to add to the Changelog.

**Comment by aconchillo** [comment]
LGTM! Just one comment.

File: src/pipecat/services/aws.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                "OutputFormat": "pcm",
+                "VoiceId": self._voice_id,
+                "Engine": self._params.engine,
+                "SampleRate": str(self._sample_rate),
+            }
+
+            # Filter out None values
+            filtered_params = {k: v for k, v in params.items() if v is not None}
+
+            response = self._polly_client.synthesize_speech(**filtered_params)
```
Comment (aconchillo) [inline src/pipecat/services/aws.py:147]: Oh! I just realized this should be in a thread like this:

```
        response = await asyncio.to_thread(self._polly_client.synthesize_speech(**filtered_params))
```

Comment (aconchillo) [reply src/pipecat/services/aws.py:147]: Otherwise it will block the asyncio event loop.

**Pull request merged**